GUI/QtQuick/QtQuickAdapter.cpp

```diff
@@ -232,17 +232,17 @@ void QtQuickAdapter::run(int &argc, char **argv)
    rootCtxt_->setContextProperty(QLatin1Literal("hwDeviceModel"), hwDeviceModel_);
    rootCtxt_->setContextProperty(QLatin1Literal("walletBalances"), walletBalances_);
    rootCtxt_->setContextProperty(QLatin1Literal("feeSuggestions"), feeSuggModel_);
-   engine.addImageProvider(QLatin1Literal("QR"), new QRImageProvider);
+   engine_->addImageProvider(QLatin1Literal("QR"), new QRImageProvider);
 
-   engine.load(QUrl(QStringLiteral("qrc:/qml/main.qml")));
-   if (engine.rootObjects().empty()) {
+   engine_->load(QUrl(QStringLiteral("qrc:/qml/Debug/main_debug.qml")));
```

Comment:
This can be wrapped around as
```
#ifndef NDEBUG
   <load debug QML>
#else
   <load release QML>
#endif
```
Also, I think it's enough to have one "debug" name, so either "/qml/Debug/main.qml" or "/qml/main_debug.qml".